scripts/variantstore/wdl/extract/populate_alt_allele_table.py

```diff
@@ -31,10 +31,11 @@ def populate_alt_allele_table(query_project, vet_table_name, fq_dataset, sa_key_
     alt_allele_temp_function = Path('alt_allele_temp_function.sql').read_text()
     alt_allele_positions = Path('alt_allele_positions.sql').read_text()
     fq_vet_table = f"{fq_dataset}.{vet_table_name}"
+    fq_sample_info = f"{fq_dataset}.sample_info"
     query_with = f"""INSERT INTO `{fq_dataset}.alt_allele`
                 WITH 
-                  position1 as (select * from `{fq_vet_table}` WHERE call_GT IN ('0/1', '1/0', '1/1', '0|1', '1|0', '1|1', '0/2', '0|2','2/0', '2|0')), 
-                  position2 as (select * from `{fq_vet_table}` WHERE call_GT IN ('1/2', '1|2', '2/1', '2|1'))"""
+                  position1 AS (SELECT vet.* FROM `{fq_vet_table}` vet JOIN `{fq_sample_info}` info ON vet.sample_id = info.sample_id WHERE info.withdrawn IS NULL AND vet.call_GT IN ('0/1', '1/0', '1/1', '0|1', '1|0', '1|1', '0/2', '0|2','2/0', '2|0')),
```

Comment:
It might be good to run these both ways (Join vs IN) and see what the bytes scanned look like.  This query touches a lot of data…

Reply:
Would putting the queries into the google console and comparing the "This query will process x when run" work?  Or does it need to actually run and then look at the `COMPLETED (X seconds, 0 retries, Y MBs) - into alt allele from vet_001` output?

Reply:
I think the google console would give you a good enough idea.  They could scan the same amount, it's just hard to guess with BQ.

Reply:
I did put both versions of this query in the console for my one quickstart dataset and the estimated amounts to be scanned were the same. However I did _not_ execute the query so I don't know how actual runtime performance compares.